src/controllers/dictionaryController.ts

```diff
@@ -73,15 +74,15 @@ export const updateSchema = async (req: Request, res: Response) => {
 
 export const diffDictionaries = async (req: Request, res: Response) => {
   const showReferences = req.query.references || false;
-  const name = req.query.name;
-  const leftVersion = req.query.left;
-  const rightVersion = req.query.right;
+  const name = req.query.name.toString();
+  const leftVersion = req.query.left.toString()
+  const rightVersion = req.query.right.toString();
```

Comment:
Same question re: will this break if the client doesn't submit one of these params?